### PR TITLE
Math masters: Add events to calculator app

### PR DESCRIPTION
**Pull request by AFakhriddinov**
**In this milestone,**

Added events 
Used props
Used states
All operations in calculated work
Changed div element to button element to avoid eslint errors

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,44 +1,100 @@
...
+      <div className="operand-input">
+        <input
+          type="text"
+          name="operand"
+          value={state.data.next || state.data.total || 0}
+        />
+      </div>
+    );
+  }
+};
```
Comment (shella12) [inline src/components/Calculator.js:99]: - [ ] [OPTIONAL] I would suggest putting your `Operand `component in a separate file from it's parent component and use `import` to access it. React is all about re-using code, and it is recommended to split your components into separate files.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,44 +1,100 @@
...
-);
+  function Operand(props) {
+    const state = props;
+    return (
+      <div className="operand-input">
+        <input
+          type="text"
+          name="operand"
+          value={state.data.next || state.data.total || 0}
+        />
```
Comment (shella12) [inline src/components/Calculator.js:95]: - [ ] [OPTIONAL] The negative sign is displayed on the wrong side for negative values. I would suggest you to rearrange the `state.data` values. Example snippet:

```js

 <div className="panel">
          {total}
          {operation}
          {next}
        </div>
```

**Review by shella12 — APPROVED** [review]
Hi @AFakhriddinov, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Click events are implemented for all buttons ✔️ 
- Math logic files are present👍 
- State handled with `useState`

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (AFakhriddinov) [reply src/components/Calculator.js:99]: Thanks for your feedback. I will consider this

File: src/components/Calculator.js (hunk 2 above)
Comment (AFakhriddinov) [reply src/components/Calculator.js:95]: Thanks again. I will fix this issues

**Pull request merged**